purchases/build.gradle

```diff
@@ -12,12 +16,12 @@ dependencies {
     implementation project(":strings")
     api project(":public")
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     implementation "androidx.annotation:annotation:$annotationVersion"
     latestDependenciesApi "com.android.billingclient:billing:$billingVersion"
     unityIAPApi "com.android.billingclient:billing:$billingClient3Version"
     implementation "androidx.lifecycle:lifecycle-runtime:$lifecycleVersion"
-    kapt "androidx.lifecycle:lifecycle-compiler:$lifecycleVersion"
+    implementation "androidx.lifecycle:lifecycle-common-java8:$lifecycleVersion"
```

Comment:
I believe you can remove the `-java8` suffix, since everything was moved to the `lifecycle-common` dependency